eth/backend.go

```diff
@@ -97,8 +97,8 @@ type Ethereum struct {
 
 	networkID     uint64
 	netRPCService *ethapi.PublicNetAPI
-
-	lock sync.RWMutex // Protects the variadic fields (e.g. gas price and etherbase)
+	NodeID        string       // @rayyuan id of node
+	lock          sync.RWMutex // Protects the variadic fields (e.g. gas price and etherbase)
```

Comment:
Do you mean "variable" fields?

Reply:
I didn't add the sync.RWMutex. It was a field from before